src/main/java/io/cdap/plugin/http/source/common/BaseHttpSourceConfig.java

```diff
@@ -276,9 +283,32 @@
   @Macro
   protected Long waitTimeBetweenPages;
 
-  @Name(PROPERTY_OAUTH2_ENABLED)
-  @Description("If true, plugin will perform OAuth2 authentication.")
-  protected String oauth2Enabled;
```

Comment:
Was it released already? If yes, do we provide backwards compatibility?

Reply:
Oauth2 was already released. I believe I only changed ways to determine if Oauth2 is enabled, the actual workflow stays the same

Reply:
If it's already released, I don't think we can take the property back. We risk not being able to read old configuration, correct me if I am wrong

Reply:
ok, it makes sense.

Reply:
But one of the issue I have is if I leave it there, the bottom of the page will show this extra element:
![image](https://user-images.githubusercontent.com/98125204/160213949-d2c3cb22-1199-41c1-9fd1-8f8c9ff25464.png)